### PR TITLE
coq-coinduction package (first release)

### PR DESCRIPTION
**Pull request by damien-pous**

### CODE REVIEW

File: released/packages/coq-coinduction/coq-coinduction.1.0/opam [hunk 1]
```diff
@@ -0,0 +1,43 @@
+opam-version: "2.0"
+maintainer: "damien.pous@ens-lyon.fr"
+version: "1.0"
```
Comment (palmskog) [inline released/packages/coq-coinduction/coq-coinduction.1.0/opam:3]:
```suggestion
```

File: released/packages/coq-coinduction/coq-coinduction.1.0/opam [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  "keyword:CCS"
+  "logpath:Coinduction"
+]
+authors: [
+  "Damien Pous"
+]
+url {
+  src:
+    "https://github.com/damien-pous/coinduction/archive/v1.0.tar.gz"
+  checksum: "md5=2ea0f8e1bb16a8ced44b33b7578ab054"
```
Comment (palmskog) [inline released/packages/coq-coinduction/coq-coinduction.1.0/opam:42]:
```suggestion
  checksum: "sha512=47d3a3dc7a2bbbd361131344b08133160353df3997a5df9bae4e6f91ee128b6d9212308ffd7709e9a0379abb4314e569353dba55062c61e0196bce0a4258e1c0"
```
We're trying to move away from MD5.

File: released/packages/coq-coinduction/coq-coinduction.1.0/opam [hunk 3]
```diff
@@ -0,0 +1,43 @@
+opam-version: "2.0"
+maintainer: "damien.pous@ens-lyon.fr"
+version: "1.0"
+
+homepage: "https://github.com/damien-pous/coinduction"
+dev-repo: "git+https://github.com/damien-pous/coinduction.git"
+bug-reports: "https://github.com/damien-pous/coinduction/issues"
+license: "LGPL-3.0-or-later"
+
+synopsis: "A library for doing proofs by (enhanced) coinduction"
```
Comment (palmskog) [inline released/packages/coq-coinduction/coq-coinduction.1.0/opam:10]:
```suggestion
synopsis: "A library and plugin for doing proofs by (enhanced) coinduction"
```
Maybe good to point out that this is a plugin as well.

File: released/packages/coq-coinduction/coq-coinduction.1.0/opam (hunk 1 above)
Comment (damien-pous) [reply released/packages/coq-coinduction/coq-coinduction.1.0/opam:3]: we are no longer supposed to indicate the version in the opam file?

Comment (palmskog) [reply released/packages/coq-coinduction/coq-coinduction.1.0/opam:3]: The version is already defined by the directory path (`coq-coinduction.1.0`). We avoid redundant version specification to make it easier to copy `opam` files.

**Pull request merged**